documentation/modules/managing/con-consumer-config-properties.adoc

```diff
@@ -65,6 +80,30 @@ group.id=my-group-id <1>
 ----
 <1> Add a consumer to a consumer group using a group id.
 
+== Choosing the right partition assignment strategy
+
+Select an appropriate partition assignment strategy, which determines how Kafka topic partitions are distributed among consumer instances in a group.
+
+Partition strategies are supported by the following classes:
+
+* `org.apache.kafka.clients.consumer.RangeAssignor`
+* `org.apache.kafka.clients.consumer.RoundRobinAssignor`
+* `org.apache.kafka.clients.consumer.StickyAssignor`
+* `org.apache.kafka.clients.consumer.CooperativeStickyAssignor`
+
+Specify a class using the `partition.assignment.strategy` consumer configuration property. 
+The *range* assignment strategy assigns a range of partitions to each consumer, and is useful when you want to process related data together.
+
+Alternatively, opt for a *round robin* assignment strategy for equal partition distribution among consumers, which is ideal for high-throughput scenarios requiring parallel processing.
+
+For more stable partition assignments, consider the *sticky* and *cooperative sticky* strategies. 
+Sticky strategies maintain assigned partitions during rebalances, when possible. 
+If a consumer was previously assigned certain partitions, the sticky strategies aim to reassign those same partitions back to the same consumer after a rebalance.
```

Comment:
IIRC it's not going to "reassign" in the sense that it revokes and reassigns, leaving the consumer not consuming until the reassignment happens. IIRC it doesn't revoke the partitions which should stick with that consumer. Just revokes the ones which will be moved to another consumer. @mimaison can you confirma that? If yes, @PaulRMellor could we make it clearer here?

Reply:
That's a fair point. I don't think reassign is a good way of wording this. I've updated here:  [docs(review): edits to doc from review by PP](https://github.com/strimzi/strimzi-kafka-operator/pull/9260/commits/f8c19a7ce3d24e6f61064f3a81b073034c3883b1)

Reply:
You fine with this @ppatierno? Can we merge it?

Reply:
Yes it's fine with me. Thanks.